src/plugins/Emojis/styling/EmojiPicker.scss

```
@@ -1,11 +1,389 @@
+    }
+
+    &__category-nav {
+      display: flex;
+      align-items: stretch;
+      gap: var(--str-chat__spacing-xxs);
+      padding: var(--str-chat__spacing-xs) var(--str-chat__spacing-sm) 0;
+    }
+
+    &__category-nav-button {
+      @include utils.button-reset;
+      cursor: pointer;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)